stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1589,42 +1623,41 @@ LogicalResult verifyConvolutionAttributes(ConvolutionOp op) {
 //  1. Input args to ConvolutionOp 'op' are RankedTypes.
 //  2. rank-of(input-type) == rank-of(output-type)
 SmallVector<int64_t> inferConvolutionOpReturnShape(
-    ConvolutionOp op, const ArrayRef<hlo::WindowDimension> window) {
-  // We keep the 'unknown' dimensions (cl/415132294) as it is in the
+    Value lhs, Value rhs, int64_t inputBatchDimension,
+    ArrayRef<int64_t> inputSpatialDimensions,
+    int64_t kernelOutputFeatureDimension, int64_t outputBatchDimension,
+    int64_t outputFeatureDimension, ArrayRef<int64_t> outputSpatialDimensions,
+    int64_t batchGroupCount, const ArrayRef<hlo::WindowDimension> window) {
+  // We keep the 'unknown' dimensions as it is in the
   // output-shape. To do that we initilize the output dimensions with the shape
   // of the return-type and updates only the spatial + non-spatial dimensions.
   // Precondition 2 ensures that size of output-shape == size of input-shape.
-  SmallVector<int64_t> outputDimensions =
-      to_vector(op.getResult().getType().cast<ShapedType>().getShape());
+  auto lhsType = lhs.getType().cast<RankedTensorType>();
+  SmallVector<int64_t> outputDimensions(lhsType.getShape().size(),
```

Comment:
Does not seem right as the unknown dimension may not mean dynamic?

Reply:
As discussed offline, we do not plan to support unknown dims in StableHLO, thus these init will all be covered by known dims. Closed this comment.